test/data/prove_consensus_gateway_proof.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Rename file name to prove_consensus_gateway.json which is aligned with delcare_open_kernel.json